### ISSUE TITLE
bug(sbb-navigation): accessibility and focusing order

### ISSUE THREAD

**Issue by jeripeierSBB** [lyne-design-system/lyne-components#1769]
Focusing order: if focus is in navigation section and you leave it, you should land on the trigger.
Only documentation: language selection should have aria-selected on it, and activated pages aria-current set to page.
